charmcraft/commands/store/registry.py

```diff
@@ -176,8 +176,8 @@ def get_manifest(self, reference):
             response = self._hit('GET', url, headers=headers)
             result = assert_response_ok(response)
             if result.get('schemaVersion') != 2:
-                raise CommandError(
-                    "Manifest v2 requested but got something else: {}".format(result))
+                logger.debug("Got something else when asking for a v2 manifest: %s", result)
+                raise CommandError("Manifest v2 not found for the indicated reference.")
             logger.debug("Got the v2 manifest ok")
```

Comment:
would it be clearer to just have "not found for {}".format(ref)

rather than "indicated reference"?

Reply:
Indeed, thanks!